nemo/core/classes/common.py

```diff
@@ -257,13 +257,21 @@ def from_config_dict(cls, config: DictConfig):
 
         if ('cls' in config or 'target' in config) and 'params' in config:
             # regular hydra-based instantiation
+            if config.cls == 'nemo.collections.asr.modules.ConvASREncoder':
+                OmegaConf.set_struct(config, False)
+                config.quantize = quantize
+                OmegaConf.set_struct(config, True)
             instance = hydra.utils.instantiate(config=config)
         elif '_target_' in config:
             # regular hydra-based instantiation
+            if config._target_ == 'nemo.collections.asr.modules.ConvASREncoder':
+                OmegaConf.set_struct(config, False)
+                config.quantize = quantize
+                OmegaConf.set_struct(config, True)
             instance = hydra.utils.instantiate(config=config)
         else:
             # models are handled differently for now
-            instance = cls(cfg=config)
+            instance = cls(cfg=config, quantize=quantize)
```

Comment:
Please revert as this will break all other nemo models since they dont support this quantize flag. There are other ways to do this.